app/assets/javascripts/react_router.js

```diff
@@ -297,6 +298,11 @@ class ReactRouter extends React.Component<Props> {
                   path="/help/keyboardshortcuts"
                   component={KeyboardShortcutView}
                 />
+                <SecuredRoute
+                  isAuthenticated={isAuthenticated}
+                  path="/timetracking"
```

Comment:
Make it a user-specific route? `/user/:user_id/timetracking`

Reply:
Why should it be user specific if the user can be selected from the dropdown? I think admins would appreciate it if they can switch between users.